test/integration/visitor_sees_pursuits_by_activities_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class VisitorSeesPursuitsByActivitiesTest < ActionDispatch::IntegrationTest
+  test "visitor can see list of activities" do
+    create_pursuits(2, "Hiking")
+    create_pursuits(2, "Fishing")
+
+    visit pursuits_path
+
+    assert page.has_content?("Hiking")
+    assert page.has_content?("Fishing")
+  end
+
+  test "visitor can see pursuits sorted by activities" do
+    create_pursuits(2, "Hiking")
+    create_pursuits(2, "Fishing")
+
+    visit pursuits_path
+    click_link "Hiking"
+
+    assert_equal '/hiking', current_path
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.